benchmarks/src/main/scala/zio/chunks/ChunkArrayBenchmarks.scala

```diff
@@ -1,7 +1,7 @@
 package zio.chunks
 
 import org.openjdk.jmh.annotations._
-import zio._
+import zio.{State => _, _}
```

Comment:
`State` is a relatively common name. We could potentially rename to `ZState` if we did not want to use `ZState` for a more polymorphic version.

Reply:
I think we have to move to `ZState` given how many clashes there will be with `State`. Alternatively, it could be something like `EnvState`.